lib/beaker/module_install_helper.rb

```diff
@@ -144,6 +167,22 @@ def get_module_source_directory(search_in)
     end
     module_source_dir
   end
+
+  def forge_host
+    fh = ENV['BEAKER_FORGE_HOST'] || ENV['forge_host']
```

Comment:
@DavidS I misread the beaker source, I was wrong. I have removed this from the code now. Thanks :)